decidim-admin/app/forms/decidim/admin/component_form.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+module Decidim
+  module Admin
+    # A form object used to create participatory processes from the admin
+    # dashboard.
+    #
+    class ComponentForm < Decidim::Form
+      include TranslatableAttributes
+
+      mimic :component
+
+      translatable_attribute :name, String
+      attribute :step_id, Integer
```

Comment:
Not sure about this actually, what if we add a method `step` that finds the step? (so other can use it instead of the id). I don't like passing ids around.